reference/toaztr.v1.yaml

```diff
@@ -464,6 +522,12 @@ components:
       properties:
         data:
           $ref: ../models/ScoreDetails.v1.json
+    PartnerBillingResponse:
```

Comment:
I don't think we need this

Reply:
I did the same way as NotaryFeesResponse or IncomeTaxResponse for instance.
It should be different here ?

Reply:
Yes it should be like /case, notaryFees is not a resource, it's only a computation.

Reply:
Ah ok, removed